main.py

```diff
@@ -0,0 +1,7 @@
+from parse_input import parse_args
+from commands.command_factory import CommandFactory
+
+if __name__ == "__main__":
+   args = parse_args()
+   CommandFactory.create_command(args)
```

Comment:
Where are you calling execute ?

Reply:
In the validate. Fisrt, the validate function is excuted and if everything is ok, the execute is called inside the validate